src/recruiting/router.js

```diff
@@ -1,14 +1,453 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
+let Student = require("./models/potential-student").PotentialStudent;
+let Teacher = require("./models/teacher").Teacher;
+let Meeting = require("./models/meeting").Meeting;
 /**
  * @swagger
  * /students:
  *    get:
  *      description: Return list of students
+ *      produces:
+ *        - application/json
+ *      responses:
+ *        201:
+ *          description: Return created students
+ *        400:
+ *          description: Something unexpected went wrong
  */
+
 router.get("/students", (req, res) => {
-    return res.send("Hello");
+  Student.find()
+    .then(students => {
+      if(students == null || students == []) {
+        res.status(200).json({"message" : "no students found"})
```

Comment:
Dit hoort niet in een REST api, gewoon een lege array returnen, je kunt er ook een shorthand if statement  van maken

`.then(students => students ? res.status(200).json(students) : res.status(200).json([])`